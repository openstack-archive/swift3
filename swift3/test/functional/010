#!/bin/bash

# Initiate MultiPart Upload

. ./common

S3USER=tester

_s3_put /bucket

_s3_post /bucket/object?uploads \
    -H "x-amz-acl: public-read-write" \
    -H "x-amz-meta-foo1: 1" \
    -H "x-amz-meta-foo2: 2" \
    -H "x-amz-meta-foo3: 3" | _filter_curl
