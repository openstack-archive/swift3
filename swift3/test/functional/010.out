QA output created by 010
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X POST -H x-amz-acl: public-read-write -H x-amz-meta-hoge1: 1 -H x-amz-meta-hoge2: 2 -H x-amz-meta-hoge3: 3 http://SWIFT_HOST/bucket/object?uploads... 200
<?xml version='1.0' encoding='UTF-8'?>
<InitiateMultipartUploadResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Bucket>bucket</Bucket><Key>object</Key><UploadId>UPLOAD_ID</UploadId></InitiateMultipartUploadResult>